hack/jenkins/preload_generation.sh

```diff
@@ -20,6 +20,9 @@
 
 set -eux -o pipefail
 
+mkdir -p cron && gsutil -qm rsync "gs://minikube-builds/${MINIKUBE_LOCATION}/cron" cron || echo "FAILED TO GET CRON FILES"
```

Comment:
```suggestion
mkdir -p cron && gsutil -qm rsync "gs://minikube-builds/master/cron" cron || echo "FAILED TO GET CRON FILES"
```

Unless you updated the job to pass in `MINIKUBE_LOCATION`.